src/mysqlsh_helpers.py

```diff
@@ -38,3 +66,38 @@ def create_cluster(self):
     def add_instance_to_cluster(self):
         """Add an instance to the InnoDB cluster."""
         pass
+
+    def run_mysqlsh_script(self, script, mode="python"):
+        """Execute a MySQL shell script
+
+        :param script: Mysqlsh script
+        :type script: str
+        :param mode: Script type (python, or sql)
+        :type mode: str
+        :side effect: Calls subprocess.check_output
+        :raises subprocess.CalledProcessError: Raises CalledProcessError if the
```

Comment:
to confirm: this is actually the docstring format on the other database operators and actually something we want to adopt